controllers/ApiController.php

```diff
@@ -706,25 +706,36 @@ public function actionChangePermissions($path = null, $item = null)
         ]);
     }
 
-    public function actionSearch($q, $limit = '')
+    /**
+     * @param string $q search string
+     * @param int $limit limit result
+     * @param bool $only_files limit result to files, default true
+     * @param int $q_limit limit items in the db query that will be checked for read access, default 1000
+     *
+     * @return Response
+     */
+    public function actionSearch($q, $limit = null, $only_files = true, $q_limit = 1000)
```

Comment:
You introduced a new parameter here but never used it. I guess this one was intended for an If condition at [this point here in Line 726](https://github.com/dmstr/yii2-filefly-module/pull/36/files#diff-1a55d252b8475f87102f6c68ee116cfe4cd7c04641d1b04e773cd2826b9f1318R726) or am  I missing something?

Reply:
Apart from that, the rest LGTM

Reply:
You got that right, thanks for the pointer. if added.